benchmarks/src/generated/main/grpc/io/grpc/benchmarks/proto/BenchmarkServiceGrpc.java

```diff
@@ -157,6 +157,12 @@ protected BenchmarkServiceFutureStub build(io.grpc.Channel channel,
     }
   }
 
+  public static abstract class AbstractBenchmarkService implements BenchmarkService, io.grpc.stub.BindableService {
```

Comment:
Internally at Google (and strongly related to #1469)  we have Abstract class versions called BaseRouteGuideService or the like, which basically protects clients from interface changes breaking their build.  The default just fails the rpc immediately while allowing the code to still compile.

Now, with that in mind, how does your change fit into this?  You have a new abstract class that does something that looks like it saves you a few lines, and is another abstract class that is in our interface.  Which abstract class is the correct one to use?  The answer isn't "just tab complete it" any more, which means now people will have to look into the generated code to see which one is right.  That cost is too much.  To add to the API, there needs to be really outstanding gains, on the order of making something impossible possible.  How does this the vast majority of people who aren't using an annotation based service binding?